_posts/2024-03-01-pyos-newsletter-march-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: March 2024"
+excerpt: "New partnerships, packages, and conference announcements!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-march-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+toc: true
+comments: true
+---
+
+## :star: Welcome, AstroPy
+
+pyOpenSci has officially partnered with the [AstroPy Project](https://www.astropy.org/) through our [Community Partnership program](https://www.pyopensci.org/partners.html). AstroPy is a Community Python Library for Astronomy, and the AstroPy Project is a community effort to develop a single core package for Astronomy in Python, while also fostering interoperability between Python astronomy packages. We're thrilled to be partnering with the AstroPy project, and can't wait to share more!
```

Comment:
What are your thoughts on graphics for blog posts? 
this graphic: 
![image](https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/ea22f458-6baa-4208-8851-7fad63104327)

could be here for instance with a caption about how community partnerships work.

Reply:
I think graphics are great! normally I'd include (many) more, but because this is our first pass at this, and because this is a back-dated post, I didn't take the time to get graphics incorporated. normally I like at least one graphic//visual per section.

Reply:
ok cool. let's plan for the future then. good point about this being backdated.